test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/GetInteriorVehicleDataConsent/004_HMI_no_response_SIVD.lua

```diff
@@ -29,6 +29,7 @@ runner.testSettings.isSelfIncluded = false
 --[[ Local Variables ]]
 --modules array does not contain "RADIO" because "RADIO" module has read only parameters
 local modules = { "CLIMATE", "AUDIO", "LIGHT", "HMI_SETTINGS" }
+local timeout = 21000
```

Comment:
Is this to simulate no response after default SDL timeout? If so, can this be read from config file instead of hardcoding it? Same for rest of test cases below as well.

Reply:
@atiwari9 Good idea. We'll move this into `common` module